linux_os/guide/system/auditing/auditd_configure_rules/audit_privileged_commands/audit_rules_privileged_commands_kmod/rule.yml

```diff
@@ -1,6 +1,6 @@
 documentation_complete: true
 
-prodtype: sle12,sle15
+prodtype: rhel8,sle12,sle15
```

Comment:
could RHCOS be added too? Doesn't need to be in this PR. Just thought it's a good idea.